habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -244,6 +244,31 @@ class BaseVelocityActionConfig(ActionConfig):
     allow_back: bool = True
 
 
+@dataclass
+class BaseVelocityNonCylinderActionConfig(ActionConfig):
+    r"""
+    In Rearrangement only for the non cylinder shape of the robot. Corresponds to the base velocity. Contains two continuous actions, the first one controls forward and backward motion, the second the rotation.
+    """
+    type: str = "BaseVelNonCylinderAction"
+    # The max linear speed of the robot
+    lin_speed: float = 10.0
```

Comment:
if we allow lateral velocity, can we add a max speed for that as well, instead of sharing lin_speed in the 2 dimensions?

Reply:
Done, thank you!